src/main/java/prefs/Config.java

```diff
@@ -0,0 +1,8 @@
+package prefs;
+
+/**
+ * An interface that represents configuration information in HubTurbo.
+ */
+public interface Config {
```

Comment:
Why do we need a marker interface? If we are not using it now, then let's not add it now. We can wait till it is needed.